.github/workflows/testflight.yml

```diff
@@ -0,0 +1,85 @@
+name: Upload to TestFlight
+on:
+  push:
+    branches:
+      - main
+      - feat/sample-test-flight
+
+env:
+  FASTLANE_KEYCHAIN_NAME: fastlane.keychain
+
+jobs:
+
+  upload_to_testflight:
+    name: Build and Upload Sample App to TestFlight
+    runs-on: macos-latest
+    working-directory: ./flutter/example
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: sudo xcode-select -s /Applications/Xcode_12.4.app/Contents/Developer
+
+      # The Github Actions hang when Xcode is trying to sign the app because
+      # a modal window pops up to access the signing key's keychain. This modal
+      # lets the action hang. A workaround is to create a local keychain and
+      # store the signing keys in there.
+      # https://github.com/actions/virtual-environments/issues/1820#issuecomment-719549887
+      
+      # Sometimes the keychain already exists
+      - name: Delete local keychain
+        continue-on-error: true
+        run: security delete-keychain $FASTLANE_KEYCHAIN_NAME
+          
+      - name: Create local keychain
+        run: |
+          security create-keychain -p ${{ secrets.FASTLANE_KEYCHAIN_PASSWORD }} $FASTLANE_KEYCHAIN_NAME
+          security set-keychain-settings -lut 21600 $FASTLANE_KEYCHAIN_NAME
+          security unlock-keychain -p ${{ secrets.FASTLANE_KEYCHAIN_PASSWORD }} $FASTLANE_KEYCHAIN_NAME
+          security list-keychain -d user -s $FASTLANE_KEYCHAIN_NAME
```

Comment:
`l`: I moved this to Fastlane in Cocoa to make it easier for local debugging. See https://github.com/getsentry/sentry-cocoa/blob/ab0326835655fbe7cbfec32a80dd1efb9876f128/fastlane/Fastfile#L46-L59